ui/imports/shared/views/AssetsView.qml

```diff
@@ -55,6 +55,9 @@ Control {
     // shows/hides list sorter
     property bool sorterVisible
 
+    property int sortValue: -1
```

Comment:
Would be nice to add in comment the expected type of that property (`SortOrderComboBox.TokenOrder`)